src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue22987.cs

```diff
@@ -0,0 +1,26 @@
+#if WINDOWS
+// In MacCatalyst, DatePicker Text color property is not working MacOS https://github.com/dotnet/maui/issues/20904
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue22987 : _IssuesUITest
+{
+	public Issue22987(TestDevice device) : base(device) { }
+
+	public override string Issue => "DatePicker Color Icon";
+
+	[Test]
+	[Category(UITestCategories.DatePicker)]
+	public void DatePickerTextandIconColorShouldWorkProperlyOnPointerOver()
+	{
+		App.WaitForElement("Label");
+
+		//need to include something like a MovePointer and MovePointerToCoordinate methods for hovering.
+
```

Comment:
Pending snapshot on Windows:
![image](https://github.com/user-attachments/assets/706b2004-511d-4d6b-91b1-03b82f673332)
Could you commit it?

Reply:
The image is incorrect. I have restricted the test for now. Once implementation PR #27477 is merged, we can add `MoveCursor` and enable the test. @jsuarezruiz

Reply:
ok